modules/social_features/social_activity/social_activity.install

```diff
@@ -199,44 +200,19 @@ function social_activity_update_8802() {
 }
 
 /**
- * Create "Featured" view mode/display for post.
- *
- * @todo: should be moved into social_post module.
+ * Revert and create "Explore Stream" views block.
  */
 function social_activity_update_8803() {
-  // Create a new post featured entity view mode.
-  EntityViewMode::create([
-    'targetEntityType' => 'post',
-    'id' => 'post.featured',
-    'status' => TRUE,
-    'label' => t('Featured'),
-  ])->save();
+  $config_file = 'views.view.activity_stream';
+  $config = drupal_get_path('module', 'social_activity') . '/config/optional/' . $config_file . '.yml';
```

Comment:
We need to make sure we use a static file in update hooks. 
The config/optional file might change before the update hook is run, using a static file in a separate folder.

e.g. config/static/views.view.activity_stream_8803.yml could already be an option. 
Or just use the update helper, i think in this case the update helper should definitely be able to do it.

Reply:
done